src/main/java/spring/data/access/repository/JDBCTemplateCart.java

```diff
@@ -0,0 +1,41 @@
+package spring.data.access.repository;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+import spring.data.access.entity.Cart;
+import spring.data.access.mapper.CartMapper;
+
+import javax.sql.DataSource;
+
+@Repository
+public class JDBCTemplateCart implements ObjectRepository<Cart> {
```

Comment:
its DAO 
should be CartDao or CartRepository
not JDBCTemplateCart

Reply:
have been renamed